elfpy/markets/hyperdrive/hyperdrive_market.py

```diff
@@ -411,6 +464,27 @@ def close_long(
             mint_time=mint_time,
         )
         self.market_state.apply_delta(market_deltas)
+
```

Comment:
same as above -- can you add a comment explaining why we want to apply deltas after checkpointing on open but before checkpointing on close?